pyfr/mputil.py

```diff
@@ -25,6 +25,16 @@ def jacobi(n, a, b, z):
     return j
 
 
+def derivative(f, x):
```

Comment:
I think mp.diff cannot work for array of function. It is easier to implement than jacobi_diff.